### PR TITLE
Add an integration test for rockcraft based OCI image build

### PR DESCRIPTION
**Pull request by weiiwang01**

### CODE REVIEW

File: tests/workflows/integration/test-rock/rockcraft.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: test
+summary: Test rock.
+description: Test rock.
+version: "1.0"
+base: ubuntu:22.04
```
Comment (arturo-seijas) [inline tests/workflows/integration/test-rock/rockcraft.yaml:5]: Perhaps using "bare" will be faster

**Review by arturo-seijas — COMMENTED** [review]
Just a minor suggestion

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by amandahla — DISMISSED** [review]
LGTM

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by arturo-seijas — DISMISSED** [review]
LGTM

**Review by arturo-seijas — APPROVED** [review]
LGTM

**Comment by merkata** [comment]
Just to be on the safe side - the logic is that in the integration workflow we skip building the rock and it is built in the integration-rock workflow, correct? Looks good!

**Review by merkata — APPROVED** [review]

**Comment by weiiwang01** [comment]
> Just to be on the safe side - the logic is that in the integration workflow we skip building the rock and it is built in the integration-rock workflow, correct? Looks good!

Yes. In the integration, the OCI image is built by docker.

**Pull request merged**